src/learn/DOTNET/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Introduction to Dotnet
+description: Learn how to set up a basic DOTNET app
+tags:
+  - workshop
+keywords:
+  - DOTNET
+  - ASP.NET 
+  - CSharp
+challenge: https://github.com/foundersandcoders/DOTNET-Workshop
+---
+
+Although all are coding so far has been in javascript or its close cousin typescript not all web sites are coded in this language. If you know other popular coding languges it will boost you capabilities as a dev.
+The benefits of languges over one another have much to do with situation and prefference. Proponents of DOTNET(.NET) will tell you that in comparison to javascript the langauge was designed with much more intention and while javascript devs are constantly installing new libraries to do anything dotnet has in its built in library most of the functionality needed to be a developer. They will also tell you that when you get to the point of maximizing the speed of your code .NET will make things easier. Many people are also fans of object oriented programing but coming from functional programing its unlikely you will initially feel that way. 
+
+## Setting up .Net
+
+Dot net projects can be made in VSCode just like javascript. In order to get started with this you will need to first install the dot net SDK https://dotnet.microsoft.com/en-us/download and the C# Dev kit https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit
+
+
+
+
+## Setting up project
+
+start by creating a new project and cding into it.
+
+use the following command to see a list of possible dotnet project types. 
+
+```shell
+dotnet new list
+```
+
+next use this command to make a ASP.NET core web api with the name MyWebApi 
+
+```dotnet
+
+dotnet new webapi -n MyWebApi
```

Comment:
have you verified this one? I only know `-o` as the "output" flag to name the file. I've never seen `-n` instead